hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/container/ContainerStateManagerV2.java

```diff
@@ -130,6 +141,21 @@ ContainerInfo getContainer(ContainerID containerID)
   void addContainer(ContainerInfoProto containerInfo)
       throws IOException;
 
+  /**
+   *
+   */
+  @Replicate
+  void updateContainerState(HddsProtos.ContainerID id,
```

Comment:
Can we add unit test if any the updateContainerState with @Replicate?

Reply:
Sure.